tests/test_checkpointing.py

```diff
@@ -0,0 +1,199 @@
+import csv
+import os
+import shutil
+import tempfile
+import unittest
+import src.trainer as trainer
+import torch
+from src import evaluate
+import os.path
+from main import evaluate_and_write
+import pandas as pd
+from src.models import MultiTaskModel
+import src.tasks.tasks as tasks
+from unittest import mock
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data import Instance, Token, vocabulary
+from ..allennlp_mods.numeric_field import NumericField
+from allennlp.data.fields import LabelField, ListField, MetadataField, TextField
+from allennlp.common.params import Params
+from allennlp.data.iterators import BasicIterator, BucketIterator
+from allennlp.training.learning_rate_schedulers import (  # pylint: disable=import-error
+    LearningRateScheduler,
+)
+from allennlp.training.optimizers import Optimizer
```

Comment:
do you mind cleaning this imports up a bit? (default libraries on top, group sources together, etc.)